cookbook/map.rst.inc

```diff
@@ -143,27 +143,30 @@
   * :doc:`/cookbook/routing/redirect_trailing_slash`
   * :doc:`/cookbook/routing/extra_information`
 
-* :doc:`/cookbook/security/index`
+* :doc:`Authentication </cookbook/security/index>`
```

Comment:
How about: `Security Authentication`

and below `Security Authorization`

so that when people search for "security", then find these